### PR TITLE
merging some footer work and fixing image alignment.

### PR DESCRIPTION
**Pull request by calee14**

### CODE REVIEW

File: lowelldevclub/templates/home.html [hunk 1]
```diff
@@ -14,20 +14,31 @@
...
 
-		<div class="center">
-			<h1 class="float-up">Lowell Dev Club</h1>
-			<p class="float-up-subtitle">Well, hello there</p>
+	<div class="center">
+		<h1 class="float-up">Lowell Dev Club</h1>
+		<p class="float-up-subtitle">Well, hello there</p>
+	</div>
+
+	<footer class="footer">
```
Comment (RafaelCenzano) [inline lowelldevclub/templates/home.html]: Footer maybe should go in base.html

File: lowelldevclub/templates/home.html [hunk 2]
```diff
@@ -14,20 +14,31 @@
 	<div class="fade-in">
 
 	<div id="particles-js"></div>
-		<nav class="navbar">
-			<a href="/" class="navbar-text">Home</a>
-			<div class="navbar-right">
-				<a href="/about" class="navbar-text" id="">About</a>
+
```
Comment (RafaelCenzano) [inline lowelldevclub/templates/home.html]: navbar should maybe go in base.html

**Comment by RafaelCenzano** [comment]
We can always move the code to base.html later

File: lowelldevclub/templates/home.html (hunk 2 above)
Comment (calee14) [reply lowelldevclub/templates/home.html]: It's kind of complicated. I've designed the navbar and footer to go with the particle js background. Maybe to match the theme of our site we can add the particle js in the background and have a container to hold the profile cards and the about content.

**Pull request merged**